internal/ent/schema/template.go

```diff
@@ -53,7 +53,7 @@ func (Template) Fields() []ent.Field {
 			Comment("the name of the template").
 			NotEmpty().
 			Annotations(
-				entgql.OrderField("name"),
+				entgql.OrderField("NAME"),
```

Comment:
Capital is only needed on enums AFAIK
```suggestion
				entgql.OrderField("name"),
```